analysis_options.yaml

```diff
@@ -1,68 +1,147 @@
 # Source of linter options:
 # http://dart-lang.github.io/linter/lints/options/options.html
 
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+
+  plugins:
+    - dart_code_metrics
```

Comment:
![](https://media.giphy.com/media/iJJ6E58EttmFqgLo96/giphy.gif)

Reply:
https://github.com/flame-engine/tiled.dart/pull/22/files#r638223887

Reply:
I rather have some few lines without trailing commas, than having my editor going out of whack because of the plugin :(